### PR TITLE
Add a link to the book page in English

### LINKED ISSUE #826: [Book] English in language list
English is not listed at Translations/Languages...
How to switch-back (easily) to the English from other languages

https://github.com/progit/progit2/issues/641

### PR DESCRIPTION
**Pull request by maxlazio**
Adds a link to the English page of the book. Fixes #826

### CODE REVIEW

File: app/views/books/_translations.html.erb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <p>
+This book is available in
+  <a href="/book/en">English</a>.
+</p>
+<p>
```
Comment (peff) [inline app/views/books/_translations.html.erb]: Makes sense. Do wording nits:

  - we now say "this book" in two consecutive paragraphs. Should it be "This book is available in English" followed by "It has been translated into..." or something?

  - is it worth mentioning that the book isn't just available in English, but that this is the primary/original language?

Comment (maxlazio) [inline app/views/books/_translations.html.erb]: Yeah, I was thinking about it but I wasn't too sure how to formulate that (the latter).

Originally I thought having something like "Pro Git is written in English" but it sounded a bit weird to me.

What do you think about:

  * This book is available in English
  * Full translation available in **
  * Partial translations available in **

Comment (peff) [inline app/views/books/_translations.html.erb]: Yeah, I think that makes sense. The fact that "English" is by itself in the first block probably makes it clear that it is the primary language.

Comment (maxlazio) [inline app/views/books/_translations.html.erb]: Ok, updated.

**Comment by peff** [comment]
Looks good. Thanks!

**Pull request merged**